hmvec/hmvec.py

```diff
@@ -99,19 +100,36 @@ def __init__(self,zs,ks,ms=None,params={},mass_function="sheth-torman",
         
         # Profiles
         self.uk_profiles = {}
+        self.pk_profiles = {}
         if not(skip_nfw): self.add_nfw_profile("nfw",numeric=nfw_numeric)
 
     def _init_cosmology(self,params,halofit):
         Cosmology._init_cosmology(self,params,halofit)
         self.Pzk = self._get_matter_power(self.zs,self.ks,nonlinear=False)
         if halofit is not None: self.nPzk = self._get_matter_power(self.zs,self.ks,nonlinear=True)
 
+    def update_param(self,param,value,halofit=None):
```

Comment:
This function does not exist on master anymore and should be removed.

Reply:
I've removed it.